documentation/src/main/asciidoc/search-dsl-predicate.asciidoc

```diff
@@ -287,7 +287,8 @@ on object fields that are known to have at least one sub-field that is never nul
 [IMPORTANT]
 ====
 For object fields with <<mapper-orm-indexedembedded-storage-nested,`NESTED` storage>>,
-the `exists` predicate needs to be wrapped in a <<search-dsl-predicate-nested,`nested` predicate>>.
+the `exists` predicate needs to be wrapped in a <<search-dsl-predicate-nested,`nested` predicate>>,
+or the <<search-dsl-predicate-nested-implicit,`nested` implicit form>> can be used.
 ====
```

Comment:
I would simply remove this admonition. It's not important information if it works out of the box without the user even thinking about it.

Reply:
It was the first thing I had done, then I changed it :)